doc/man3/BN_set_bit.pod

```diff
@@ -33,8 +33,10 @@ error occurs if B<a> is shorter than B<n> bits.
 BN_is_bit_set() tests if bit B<n> in B<a> is set.
 
 BN_mask_bits() truncates B<a> to an B<n> bit number
-(C<a&=~((~0)E<lt>E<lt>n)>).  An error occurs if B<a> already is
-shorter than B<n> bits.
+(C<a&=~((~0)E<lt>E<lt>n)>). An error occurs if B<a> is already shorter than
+B<n> bits or if B<n> is negative. Note that this bit check depends on the
+platform's word size. Use L<BN_num_bits(3)> to determine the exact number of
+bits if needed.
```

Comment:
```suggestion
(C<a&=~((~0)E<lt>E<lt>n)>). An error occurs if B<n> is negative. Error is also
returned if the internal representation of B<a> is already shorter than B<n>
bits. The internal representation depends on the platform's word size and the
error can be safely ignored. Use L<BN_num_bits(3)> to determine the exact number
of bits if needed.
```

Reply:
I appreciate the clearer wording in your version regarding the internal representation. However, as @nhorman pointed out, while the error about the bit size being shorter can be ignored, an error caused by a negative bit size cannot be. Therefore, I integrated your additional clarification about the internal representation but added a comma and the word “this” to make it clear that the ignorable error refers only to the second condition and not to the negative bit size.

```
BN_mask_bits() truncates B<a> to an B<n> bit number
(C<a&=~((~0)E<lt>E<lt>n)>). An error occurs if B<n> is negative. An error is
also returned if the internal representation of B<a> is already shorter than
B<n> bits. The internal representation depends on the platform's word size, and
this error can be safely ignored. Use L<BN_num_bits(3)> to determine the exact
number of bits if needed.
```